pom.xml

```diff
@@ -37,6 +37,11 @@
 
 
   <dependencies>
+    <dependency>
+      <groupId>org.commonjava.util</groupId>
```

Comment:
Sorry, this had been changed to `org.commonjava.indy.service`, aligning with the service ones.

Reply:
K. Will change it.